LabHub.toml

```diff
@@ -40,6 +40,8 @@ gitlab_repo = "brndnmtthws-oss/conky"
 [pull_request_event_trigger_actions]
 # list of enabled trigger actions
 enabled_trigger_actions = [
-    "review_requested",
     "synchronize",
+    "opened",
+    "reopened",
+    "closed",
```

Comment:
To me, this list looks totally fine. When we submit the PR to the original repo, it might be possible that they request us to add the complete list here in order to not cause a different behavior than before.

But let's wait for their review and see what they say.

Reply:
Alright, you are right. We need to put everything in here for the original repo.

Reply:
But let's just wait for their reaction first. To me, this new behavior makes more sense.